pkg/process/procutil/process_model.go

```diff
@@ -13,19 +13,19 @@ import (
 
 // Process holds all relevant metadata and metrics for a process
 type Process struct {
-	Pid      int32
-	Ppid     int32
-	NsPid    int32 // process namespaced PID
-	Name     string
-	Cwd      string
-	Exe      string
-	Comm     string
-	Cmdline  []string
-	Username string // (Windows only)
-	Uids     []int32
-	Gids     []int32
-
-	Stats *Stats
+	Pid            int32
+	Ppid           int32
+	NsPid          int32 // process namespaced PID
+	Name           string
+	Cwd            string
+	Exe            string
+	Comm           string
+	Cmdline        []string
+	Username       string // (Windows only)
+	Uids           []int32
+	Gids           []int32
+	ProcessContext []string
```

Comment:
IMO we shouldn't add this to the procutil model. This data is not collected by the procutil package, and this field is not used in the actual agent, only in the tests that you added (which can be done another way).

Reply:
Modified test to not rely on procutil modification